src/CVRF/cvrf_enumeration.c

```diff
@@ -75,7 +75,7 @@ const struct oscap_string_map CVRF_DOC_PUBLISHER_TYPE_MAP[] = {
 	{CVRF_DOC_PUBLISHER_COORDINATOR, "Coordinator"},
 	{CVRF_DOC_PUBLISHER_USER, "User"},
 	{CVRF_DOC_PUBLISHER_OTHER, "Other"},
-	{0, NULL}
+	{CVRF_ATTRIBUTE_UNKNOWN, NULL}
```

Comment:
can't mix enums like this, that will make the code unreadable. Every enum has to have it's own unknown enum value if it's necessary.